custom/icds_reports/tasks.py

```diff
@@ -1630,7 +1630,7 @@ def _get_docs_in_ucr(domain, table_id, doc_ids):
             WHERE doc_id = ANY(%(doc_ids)s);
         '''
         cursor.execute(query, {'doc_ids': doc_ids})
-        return dict(cursor.fetchall())
```

Comment:
previously the query returned tuples of `(doc_id, inserted_at)` so `dict` was creating a dictionary of doc_id -> inserted_at. This broke when I changed it to only returning `doc_id`